test cases/common/139 simd/meson.build

```diff
@@ -0,0 +1,31 @@
+project('simd', 'c')
+
+simd = import('simd')
+
+cc = meson.get_compiler('c')
+
+cdata = configuration_data()
+
+simdlibs = simd.check('mysimds',
```

Comment:
For instance, they might want to use it in multiple targets that are built with different compilers.